proto/stream_plan.proto

```diff
@@ -577,8 +576,6 @@ message StreamActor {
   // It is painstaking to traverse through the node tree and get upstream actor id from the root StreamNode.
   // We duplicate the information here to ease the parsing logic in stream manager.
   repeated uint32 upstream_actor_id = 6;
-  // Placement rule for actor, need to stay on the same node as a specified upstream actor.
-  ColocatedActorId colocated_upstream_actor_id = 7;
```

Comment:
The field is here because the `StreamActor` message is a protocol between the old graph builder and the old scheduler. As we remove the old scheduler and schedule fragments ahead of time, we can remove this.